### ISSUE TITLE
please give us a complete tutorial

### ISSUE THREAD

**Issue by maxsens026** [EugeneSunrise/reWASD#9]
we cant turn remap on please help us show us tutorial

**Comment by EugeneSunrise** [comment]
Waiting for more feedback, as some people have everything working and some not working

**Comment by bravazeroh** [comment]
1. If you have any version of reWASD installed, uninstall it using control panel.
2. Restart PC
3. In releases install reWASD-700-8447.exe
4. Follow the instructions during the wizard
5. Restart PC again
6. In releases, install reWASD.dll and reWASDCommon.dll
7. Then move the two DLLs you installed in to reWASD main folder which for me, is in C:\Program Files\reWASD
8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!

**Comment by maxsens026** [comment]
I did everything according to your instructions but still not working

**Comment by maxsens026** [comment]
I don't know what you mean Install dll files I just copy them

**Comment by maxsens026** [comment]
![Untitled](https://github.com/EugeneSunrise/reWASD/assets/153292979/61eb11ae-3e17-4471-89fb-52151b56a471)

**Comment by maxsens026** [comment]
but the magic button is off and when i click on apply to still just blink and not working

**Comment by EugeneSunrise** [comment]
> ![Untitled](https://github.com/EugeneSunrise/reWASD/assets/153292979/61eb11ae-3e17-4471-89fb-52151b56a471)
> 
> 

Properties of rewasd.dll and rewasdCommon.dll, two screenshots bro, right click on dll and properties

**Comment by bravazeroh** [comment]
> ![Untitled](https://private-user-images.githubusercontent.com/153292979/289114144-61eb11ae-3e17-4471-89fb-52151b56a471.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIwNDk0NTcsIm5iZiI6MTcwMjA0OTE1NywicGF0aCI6Ii8xNTMyOTI5NzkvMjg5MTE0MTQ0LTYxZWIxMWFlLTNlMTctNDQ3MS04OWZiLTUyMTUxYjU2YTQ3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwOFQxNTI1NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMDYzMGVlNjM3YjIzMDNjNjk3NjFhN2MzMGU5NzU5YTUzOWFkNzYzMjg2ZDk3NzlmMmQ3M2UzMWFhNWFiZDA3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ULReBCS7CqkvjXZtzrtwjWaC2BA6YhHojmsp8AsBgjI)

Why is date modified not same for all files? You need to install reWASD from Releases.. it's the .exe file

**Comment by EugeneSunrise** [comment]
> ![Untitled](https://github.com/EugeneSunrise/reWASD/assets/153292979/6c73c15d-2d2e-4bd3-8d37-3c14da79f4c4)
> 
> ![Untitled2](https://github.com/EugeneSunrise/reWASD/assets/153292979/b45a0b28-0e8d-478f-8da6-e2b990ef61bb)
> 
> 

Try to check the Unblock checkbox and press Apply button

**Comment by andrewCOL1** [comment]
> > ![Untitled](https://private-user-images.githubusercontent.com/153292979/289115403-6c73c15d-2d2e-4bd3-8d37-3c14da79f4c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIwNTk2NjcsIm5iZiI6MTcwMjA1OTM2NywicGF0aCI6Ii8xNTMyOTI5NzkvMjg5MTE1NDAzLTZjNzNjMTVkLTJkMmUtNGJkMy04ZDM3LTNjMTRkYTc5ZjRjNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwOFQxODE2MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMjk4ZmYyODRhYWJhYmQ2NGI4Yzg3NTFkMmI1YzZjNWIzMTI0MjdhOTNmOWY2ZWMxOWE2MzQyZGQ4NjEwZTU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.UA840nj_vvkCnIQrusZcC_EAja4FjQm-kRbl2lc1LEE)
> > ![Untitled2](https://private-user-images.githubusercontent.com/153292979/289115422-b45a0b28-0e8d-478f-8da6-e2b990ef61bb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIwNTk2NjcsIm5iZiI6MTcwMjA1OTM2NywicGF0aCI6Ii8xNTMyOTI5NzkvMjg5MTE1NDIyLWI0NWEwYjI4LTBlOGQtNDc4Zi04ZGE2LWUyYjk5MGVmNjFiYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwOFQxODE2MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MGI1OWEwNTg1NTA3ZDYyMDgwNDk1OWM5Njc4YTY3NWJlNjI3NTdhYmY3MGIwOGExYzMwYWMyYTU2NTE0ZGFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Rg8MptntLyBOFScXIPj2g-C2zHHTAj5d-1Z6hcQGn5E)
> 
> Try to check the Unblock checkbox and press Apply button

I have the same problem and it didn't help.

**Comment by titeta231** [comment]
> but the magic button is off and when i click on apply to slot still just blink and not working

I have this same issue

**Comment by SadewJay** [comment]
> but the magic button is off and when i click on apply to slot still just blink and not working

mine too

**Comment by u8Jj3** [comment]
This is working perfectly for me with no issue so far.



> 1. If you have any version of reWASD installed, uninstall it using control panel.
> 
>     2. Restart PC
> 
>     3. In releases install reWASD-700-8447.exe
> 
>     4. Follow the instructions during the wizard
> 
>     5. Restart PC again
> 
>     6. In releases, download reWASD.dll and reWASDCommon.dll
> 
>     7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> 
>     8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
>        This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.

I did these steps. 

I'm using Windows 10 64bit (XOS)
Note that I never installed ReWASD on that Windows installation.

**Comment by titeta231** [comment]
> This is working perfectly for me with no issue so far.
> 
> > 1. If you have any version of reWASD installed, uninstall it using control panel.
> >    
> >    2. Restart PC
> >    3. In releases install reWASD-700-8447.exe
> >    4. Follow the instructions during the wizard
> >    5. Restart PC again
> >    6. In releases, download reWASD.dll and reWASDCommon.dll
> >    7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> >    8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
> >       This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.
> 
> I did these steps.
> 
> I'm using Windows 10 64bit (XOS) Note that I never installed ReWASD on that Windows installation.

I think the problem is that I've installed rewasd before using this after my trial ended

still waiting for an update from the creator abt this situation

**Comment by Ultra-Shadeed** [comment]
someone help me with this please i need to find a cracked rewasd fast - discord: sbbab

**Comment by JugadveerSingh** [comment]
> 1. If you have any version of reWASD installed, uninstall it using control panel.
> 2. Restart PC
> 3. In releases install reWASD-700-8447.exe
> 4. Follow the instructions during the wizard
> 5. Restart PC again
> 6. In releases, download reWASD.dll and reWASDCommon.dll
> 7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> 8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
>    This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.

Did everything but no luck 😔

**Comment by Protoup20** [comment]
> > 1. If you have any version of reWASD installed, uninstall it using control panel.
> > 2. Restart PC
> > 3. In releases install reWASD-700-8447.exe
> > 4. Follow the instructions during the wizard
> > 5. Restart PC again
> > 6. In releases, download reWASD.dll and reWASDCommon.dll
> > 7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> > 8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
> >    This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.
> 
> Did everything but no luck 😔


lets try a little test, use this tut to reset the license and redo everything to see if you get the full version (if it works, wait a week just to be sure)

https://www.youtube.com/watch?v=JtLA7sb5Pao

**Comment by Doosnm** [comment]
> > > 1. If you have any version of reWASD installed, uninstall it using control panel.
> > > 2. Restart PC
> > > 3. In releases install reWASD-700-8447.exe
> > > 4. Follow the instructions during the wizard
> > > 5. Restart PC again
> > > 6. In releases, download reWASD.dll and reWASDCommon.dll
> > > 7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> > > 8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
> > >    This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.
> > 
> > 
> > Did everything but no luck 😔
> 
> lets try a little test, use this tut to reset the license and redo everything to see if you get the full version (if it works, wait a week just to be sure)
> 
> https://www.youtube.com/watch?v=JtLA7sb5Pao

its works for me! ty!

**Comment by scanlan897** [comment]
> > > > 1. If you have any version of reWASD installed, uninstall it using control panel.
> > > > 2. Restart PC
> > > > 3. In releases install reWASD-700-8447.exe
> > > > 4. Follow the instructions during the wizard
> > > > 5. Restart PC again
> > > > 6. In releases, download reWASD.dll and reWASDCommon.dll
> > > > 7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> > > > 8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
> > > >    This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.
> > > 
> > > 
> > > Did everything but no luck 😔
> > 
> > 
> > lets try a little test, use this tut to reset the license and redo everything to see if you get the full version (if it works, wait a week just to be sure)
> > https://www.youtube.com/watch?v=JtLA7sb5Pao
> 
> its works for me! ty!

did you reset the license?

**Comment by Doosnm** [comment]
> > > > > 1. If you have any version of reWASD installed, uninstall it using control panel.
> > > > > 2. Restart PC
> > > > > 3. In releases install reWASD-700-8447.exe
> > > > > 4. Follow the instructions during the wizard
> > > > > 5. Restart PC again
> > > > > 6. In releases, download reWASD.dll and reWASDCommon.dll
> > > > > 7. Then move the two DLLs you downloaded in to reWASD main folder which for me, is in C:\Program Files\reWASD
> > > > > 8. IMPORTANT!!! REPLACE THE DLL FILES WITH THE ONES YOU DOWNLOADED FROM ASSETS!!
> > > > >    This is how I installed mine (i followed EugeneSunrise's instructions bruh) Win 11 22h2 x64-based.
> > > > 
> > > > 
> > > > Did everything but no luck 😔
> > > 
> > > 
> > > lets try a little test, use this tut to reset the license and redo everything to see if you get the full version (if it works, wait a week just to be sure)
> > > https://www.youtube.com/watch?v=JtLA7sb5Pao
> > 
> > 
> > its works for me! ty!
> 
> did you reset the license?

I had the version with the expired trial period installed and a number of functions did not work for me, I performed all the actions indicated in the video and everything worked

**Comment by EugeneSunrise** [comment]
Chill and wait fix

**Comment by EugeneSunrise** [comment]
CHECK README AND GIVE ME FEEDBACK

**Comment by EugeneSunrise** [comment]
Check releases and give me feedback

**Comment by EugeneSunrise** [comment]
6.6 ver ready, 7.0 will be later, any questions? Create NEW issue
